### ISSUE TITLE
add when creating new videos, be careful about movement

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-media-guide#119]
We say in <https://www.w3.org/WAI/media/av/>:
"Some people **cannot focus and comprehend auditory or visual information** _when there are changing visuals_. For most videos, they also need descriptive transcripts."

We should add something about distracting visuals in <https://www.w3.org/WAI/media/av/av-content/>

Maybe expand on <https://www.w3.org/WAI/media/av/av-content/#avoid-causing-seizures--storyboarding-post-production-wcag-a> "Avoid causing seizures ... 
Avoid anything that flashes more than three times in any one second period."

**Comment by slhenry** [comment]
related issue in Making Presentations Accessible resource: https://github.com/w3c/wai-presentations2all/issues/16

**Comment by slhenry** [comment]
In Make presentations accessible, we say:

> **Consider how to use motion or animations**
>This includes text or images flying in from the side. Will the motion make the information easier to understand, or is it unnecessary? Certain types of motion can be particularly distracting for some people, and can make some people ill.

**Comment by AndrewArch** [comment]
@slhenry suggested "This includes text or images flying in from the side." There are other ways text/images can move onto a slide (PowerPoint offers dozens of variations including zoom, swivel and bounce), so I think we should add something like "This includes text or images flying in from the side, or otherwise move onto the screen through animation."

**Comment by nitedog** [comment]
I agree with the overall idea of the proposal. However, terms like "changing visuals", "motion", and "animation" may be understood differently in the context of video (aka ''motion pictures') than in the context of web pages and presentation slides etc. Maybe saying things like "in particular for talking-head videos" or such, to help describe the types of videos we primarily address with this type of advice.
